WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/StorePlanBannerPresenter.swift

```diff
@@ -174,20 +159,10 @@ private extension StorePlanBannerPresenter {
         onLayoutUpdated(.zero)
         self.storePlanBanner = nil
     }
-
-    /// Shows a view for the merchant to upgrade their site's plan.
-    ///
-    func showUpgradesView() {
-        guard let viewController else { return }
-
-        UpgradesViewPresentationCoordinator().presentUpgrades(for: siteID, from: viewController)
```

Comment:
Not blocking this PR. 

I see that you have logged a couple of issues like the following issue https://github.com/woocommerce/woocommerce-ios/issues/12393. Please help me understand what needs to be done to clean up this code. 

1. Should we delete the `UpgradesViewPresentationCoordinator` completely? (In a future cleanup PR)
2. I also see Free trial expiry-related local notifications. Should we delete all the "Free trial" related code?  How is Android dealing with this?

Reply:
We discussed no. 1 in Slack.

Good point about local notifications. I'm making a separate issue to check https://github.com/woocommerce/woocommerce-ios/issues/12413